usb/libusbclient/imxrt-usbc/phy.c

```diff
@@ -46,8 +46,8 @@ enum {
  * defined below, uncached map should be aligned to 0x800 both base and
  * end addresses
  */
-#define UNCACHED_BASE_ADDR     0x20020000
-#define UNCACHED_END_ADDR      0x20028000
+#define UNCACHED_BASE_ADDR 0x20050000
+#define UNCACHED_END_ADDR  0x20058000
```

Comment:
This change seems unrelated. Maybe this should be in separate PR/commit at least?

Reply:
I can put it in a different commit.
This change is needed in order to run USB client applications (e.g. cdc-demo) on imxrt without interfering with other processes, which use dtcm. The dtcm has been expanded recently in order to allow more processes to use it. Using hardcoded memory addresses is temporary, I hope.